wafer/schedule/admin.py

```diff
@@ -187,6 +189,13 @@ def changelist_view(self, request, extra_context=None):
 
 
 class SlotAdminForm(forms.ModelForm):
+
+    additional = forms.IntegerField(min_value=0, max_value=30, required=False,
```

Comment:
Will this show up when a slot is being edited? If so, is there any easy way to hide it?